source/api/plugins/browser-launch-api.md

```diff
@@ -256,6 +256,8 @@ Change the download directory of files downloaded during Cypress tests.
 
 ```js
 // cypress/plugins/index.js
+const path = require('path')
```

Comment:
lol

Reply:
true story, see support channel. I got to add a recipe where you click on the button, download Excel file and then validate it - seems it will be the most popular recipe BY FAR